### PR TITLE
#596, #653 | Send token fixes

### LINKED ISSUE #596: Successful transfer page UI improvements
## Overview
The send tokens confirmation modal could be further improved.
Screenshot 2023-02-17 at 12 48 08

## Acceptance criteria
* Move close button to the right so its similar to the other modals
* Add visual indication of the transaction being clickable
* Change text color to white so it easier to read

### PR DESCRIPTION
**Pull request by EJSG**
# Fixes #596 and #653

## Description
This PR updates the UI for successful token send state and fixes the inability to send tokens. eslint warnings are fixed in another PR, ignore them

## Test scenarios
- go to https://deploy-preview-656--open-block-explorer.netlify.app/
- sign into an account with nonzero liquid balance using anchor
- go to your account page
- click `Send`
- try to send some TLOS to another account
    - sending should work
    - the success state should match the acceptance criteria 

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file
-   [x] I have created a new issue with the required translations for the currently supported languages

### CODE REVIEW

File: src/store/account/actions.ts [hunk 1]
```diff
@@ -177,7 +177,7 @@ export const actions: ActionTree<AccountStateInterface, StateInterface> = {
                 data: data as unknown,
             },
         ];
-        return this.dispatch('sendTransaction', actions);
+        return this.dispatch('account/sendTransaction', actions);
     },
     async sendTransaction({ commit, state }, actions: Action[]) {
         console.log('sendTransaction', actions);
```
Comment (karynemayer) [inline src/store/account/actions.ts:183]:
```suggestion
```

File: src/components/SendDialog.vue [hunk 2]
```diff
@@ -251,8 +252,11 @@ export default defineComponent({
                     <q-card-section v-if="transactionId">
                         <div class="row">
                             <div class="col-12">
-                                <div class="row">You successfully sent {{ sendAmount }} {{ sendToken?.symbol }} to {{ receivingAccount }}.</div>
-                                <div class="row ellipsis-overflow" @click="navToTransaction">Click to view transaction: {{ transactionId }}</div>
+                                <div>You successfully sent {{ sendAmount }} {{ sendToken?.symbol }} to {{ receivingAccount }}.</div>
```
Comment (karynemayer) [inline src/components/SendDialog.vue:255]: I know that this is outside this issue's scope, but it feels a bit strange that the transaction result is not a standalone component. Is this by design or something that could be considered tech debt?

Comment (EJSG) [reply src/components/SendDialog.vue:255]: yeah... i would say tech debt

Comment (karynemayer) [reply src/components/SendDialog.vue:255]: For reference, I created the issue https://github.com/telosnetwork/open-block-explorer/issues/659

**Review by karynemayer — CHANGES_REQUESTED** [review]
The transaction was successful, but when clicking on the link for the transaction, I'm redirected to the main page.

**Review by donnyquixotic — APPROVED** [review]

**Pull request merged**